pkg/dataobj/internal/sections/logs/logs_test.go

```diff
@@ -82,7 +82,9 @@ func Test(t *testing.T) {
 	for result := range logs.Iter(context.Background(), dec) {
 		record, err := result.Value()
 		require.NoError(t, err)
-		actual = append(actual, record)
+		next := record.DeepCopy()
+		next.MdValueCaps = nil
```

Comment:
Why does the caller need to zero out MdValueCaps? 🤔

Reply:
This is just for testing, but `require.Equal` also compares the MdValueCaps values. I don't want the test to depend on them because they are an implementation detail, so I zero them out. 
I'm happy to move them to a helper method which ignores them. They would also disappear if we remove strings, so I'll trial that first.